sklearn/metrics/tests/test_pairwise.py

```diff
@@ -1273,3 +1273,27 @@ def test_pairwise_distances_data_derived_params(n_jobs, metric, dist_function,
 
         assert_allclose(dist, expected_dist_explicit_params)
         assert_allclose(dist, expected_dist_default_params)
+
+
+@pytest.mark.parametrize("metric", ["seuclidean"])
+@pytest.mark.parametrize(
+        "dtype",
+        [np.half, np.float, np.double, np.longdouble])
+@pytest.mark.parametrize("y_is_x", [True, False], ids=["Y is X", "Y is not X"])
+def test_pairwise_distances_input_datatypes(metric, dtype, y_is_x):
+    # check that pairwise distances gives the same result as pdist and cdist
+    # regardless of input datatype
```

Comment:
Please mention that the `seuclidean` metric used to raise an error on non double dtype, and add the PR number.

Reply:
Also fixed in recent commit